### PR TITLE
tomee-plume 8.0.5

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> Testing tomee-plume
/usr/bin/sandbox-exec -f /private/tmp/homebrew20201127-14250-m6mym8.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/tomee-plume.rb --verbose
==> /usr/local/opt/tomee-plume/libexec/bin/configtest.sh
The operation couldn’t be completed. Unable to locate a Java Runtime.
Please visit http://www.java.com for information on installing Java.

Neither the JAVA_HOME nor the JRE_HOME environment variable is defined
At least one of these environment variable is needed to run this program
Error: tomee-plume: failed
An exception occurred within a child process:
```

**Comment by chenrui333** [comment]
```
==> Testing tomee-plume
/usr/bin/sandbox-exec -f /private/tmp/homebrew20201203-57828-1jyqj9s.sb ruby -W0 -I $LOAD_PATH -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/tomee-plume.rb --verbose
==> /usr/local/opt/tomee-plume/libexec/bin/configtest.sh
Unable to find any JVMs matching version "(null)".
No Java runtime present, try --request to install.
Neither the JAVA_HOME nor the JRE_HOME environment variable is defined
At least one of these environment variable is needed to run this program
Error: tomee-plume: failed
An exception occurred within a child process:
  BuildError: Failed executing: /usr/local/opt/tomee-plume/libexec/bin/configtest.sh
```

File: Formula/tomee-plume.rb [hunk 1]
```diff
@@ -21,7 +23,8 @@ def install
     # Install files
     prefix.install %w[NOTICE LICENSE RELEASE-NOTES RUNNING.txt]
     libexec.install Dir["*"]
-    bin.install_symlink "#{libexec}/bin/startup.sh" => "tomee-plume-startup"
+    bin.env_script_all_files(libexec/"bin", Language::Java.overridable_java_home_env)
+    bin.install_symlink "#{bin}/startup.sh" => "tomee-plume-startup"
```
Comment (SMillerDev) [inline Formula/tomee-plume.rb:27]: You should fix the caveat and test then

**Comment by chenrui333** [comment]
checkin

**Pull request closed without merging**